tests/ut/test_decorators.py

```diff
@@ -414,8 +414,7 @@ def test_get_cache_keys_empty_list(self, decorator):
         assert decorator.get_cache_keys(stub_dict, (), {"keys": []}) == ([], [], -1)
 
     def test_get_cache_keys_missing_kwarg(self, decorator):
-        with pytest.raises(KeyError):
-            assert decorator.get_cache_keys(stub_dict, (), {})
+        assert decorator.get_cache_keys(stub_dict, (), {}) == ([], [], -1)
```

Comment:
this is a result of the change of inspect.signature now parsing default arguments correctly as it doesn't fail anymore